core/drivers/clk/clk-stm32mp13.c

```diff
@@ -2661,3 +2663,26 @@ static TEE_Result stm32mp13_clk_probe(const void *fdt, int node,
 }
 
 CLK_DT_DECLARE(stm32mp13_clk, "st,stm32mp13-rcc", stm32mp13_clk_probe);
+static TEE_Result stm32mp13_rcc_mco_probe(const void *fdt, int node,
```

Comment:
I prefer to managed MCO and critical clock in a sub node 
generic for each SOC of STM32MP family

=> no need to have a specific new compatible "st,stm32mp13-rcc-mco" only managed in OP-TEE for this feature

I will propose something soon for this part.

Patrick

Reply:
Ok, I understand. I'll remove the MCO clock part from the series.